bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/channeltypes.xml

```diff
@@ -229,6 +265,10 @@
 		<item-type>Number</item-type>
 		<label>Ventilation Power</label>
 		<description>Current ventilation power</description>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Power</tag>
```

Comment:
Tbh, I'm not exactly sure what this represents, but the channel is not UoM, just plain Number. It might be a level, but I don't think it's a measurement of power in terms of W, for example.

Reply:
Deleted.